tools/gulp/tasks/screenshots.ts

```diff
@@ -75,49 +95,47 @@ function getLocalScreenshotFiles(dir: string): string[] {
 }
 
 /**
+ * Get processed secure token. The jwt token is connected by '.', while '.' is not a valid
+ *   path in firebase database. Replace all '.' to '/' to make the path valid
+ */
+function getSecureToken() {
+  return process.env['FIREBASE_ACCESS_TOKEN'].replace(/[.]/g, '/');
```

Comment:
I assume this is kind of a workaround to pass the token to the Firebase functions right?

It took me some time to actually understand that it is intentionally to make it look like the Firebase child?

If so, I'd love to see a comment about this.